app/src/app/[lng]/data/[step]/SourceDrawer.tsx

```diff
@@ -63,62 +81,222 @@ export function SourceDrawer({
                 </Tag>
                 <Tag>
                   <TagLeftIcon
-                    as={MdCalendarToday}
+                    as={MdToday}
                     boxSize={6}
                     color="content.tertiary"
                   />
-                  <TagLabel fontSize={14}>
+                  <TagLabel>
                     {t("updated-every")}{" "}
                     {source.frequencyOfUpdate == "annual"
                       ? t("year")
-                      : t(source.frequencyOfUpdate)}
+                      : t(source.frequencyOfUpdate ?? "unknown")}
+                  </TagLabel>
+                </Tag>
+                <Tag>
+                  <TagLeftIcon
+                    as={MdOutlineTimer}
+                    boxSize={6}
+                    color="content.tertiary"
+                  />
+                  <TagLabel>
+                    {source.startYear} - {source.endYear}
                   </TagLabel>
                 </Tag>
               </Flex>
+
               <Stack className="space-y-4">
-                <Heading size="sm">{t("sources")}</Heading>
+                <Heading size="sm">{t("inside-dataset")}</Heading>
                 <Text color="content.tertiary">{source.description}</Text>
-                {/*
-                <Text color="content.tertiary" ml={6}>
-                  <ul>
-                    {source.sources.map((source) => (
-                      <li key={source}>{source}</li>
-                    ))}
-                  </ul>
-                </Text>
-                */}
-                <Heading size="sm">{t("methodology")}</Heading>
-                <Text color="content.tertiary">
+                <chakra.hr borderColor="border.neutral" />
+                <Heading
+                  size="sm"
+                  verticalAlign="baseline"
+                  lineHeight="24px"
+                  fontSize="16px"
+                >
+                  {t("methodology")}
                   <Link
                     href={source.methodologyUrl}
                     target="_blank"
                     rel="noreferrer noopener"
                   >
-                    {source.methodologyUrl}
+                    <Icon
+                      as={MdOpenInNew}
+                      boxSize={6}
+                      color="content.link"
+                      mb="-5px"
+                      ml={2}
+                    />
                   </Link>
+                </Heading>
+                {/*<Text color="content.tertiary">
+                  // TODO add methodology description to data source model/ data catalog
+                </Text>*/}
+                <Heading
+                  size="sm"
+                  verticalAlign="baseline"
+                  lineHeight="24px"
+                  fontSize="16px"
+                >
+                  {t("transform-data-heading")}
+                  <Link
+                    href="https://citycatalyst.openearth.org"
```

Comment:
Fair, this isn't quite the correct link yet since we don't have a methodology page. I've linked to the CC landing page instead of the app itself here.